client/src/components/ui/itemrow/util.ts

```diff
@@ -0,0 +1,5 @@
+import { Document } from "~model/document";
+
+export function redirectHandler(window: Window, id: Document['id']) {
+    window.location.href = '/track?id=' + id;
+}
```

Comment:
No need for the `window` prefix here. Also, I believe `goToTrackingPage` is a clearer name for this. 👍 
```suggestion
export function goToTrackingPage(id: Document['id']) {
    location.href = '/track?id=' + id;
}
```